packages/math/base-elements.lua

```diff
@@ -1044,7 +1130,21 @@ function elements.text:output (x, y, line)
    -- There should be no stretch or shrink on the width of a text
    -- element.
    local width = self.width.length
-   SILE.outputter:drawHbox(self.value, width)
+   -- HACK: For stretchy operators, MathML Core and OpenType define how to build large glyphs
```

Comment:
Do we have a dedicated tracking issue for someday addressing this?

Reply:
Not yet, hence my bold **workaround** notes here in the list above, and some bold too in the parent Epic (fixed). If time allows at the end of the journey, I'll want to have a refactoring session, ideally.

Reply:
#2155